TreeTracker/UI/Views/Profile Screen/ProfileViewModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ProfileViewModel.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 3/23/21.
+//  Copyright © 2021 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+protocol ProfileViewModelCoordinatorDelegate: class {
+    func profileViewModel(_ profileViewModel: ProfileViewModel, changeUser planter: Planter)
```

Comment:
`func profileViewModel(_ profileViewModel: ProfileViewModel, didLogoutPlanter planter: Planter)` would be consistent with the rest of the app

Reply:
changed the function name as didLogoutPlanter